### ISSUE TITLE
Update the CloudWorkspaceHandler to retrieve credit usage from billing vendor 

### ISSUE THREAD

**Issue by lmossman** [airbytehq/airbyte#8509]
## Tell us about the problem you're trying to solve
This is a ticket within the Integrate backend with billing vendor Epic

At this point in the epic, we should be tracking credit usage within the billing vendor, and we should have validated that there are no inconsistencies between the balances in our system and the billing vendor's system.

Therefore, it is now time to make our Cloud UI retrieve credit usage information from the billing vendor instead of from our cloud db, effectively deprecating our home-grown credit management logic.

## Describe the solution you’d like
The CloudWorkspaceHandler's `getWorkspaceUsage` endpoint should make API requests to the billing vendor to retrieve daily credit consumption information, grouped by connector/source/destination ID as necessary, rather than retrieving it from the cloud db.

**Comment by cgardens** [comment]
> We may want to instead have the source/destination/connector ID be a parameter for this API method, so that we don't need to fetch all of these aggregated usages every time this endpoint is called.

I'm not sure I fully understand this. Don't we need all of that information for the UI to render?

**Comment by lmossman** [comment]
@cgardens that comment was referring to the fact that the CloudWorkspaceHandler's `getWorkspaceUsage()` endpoint currently returns usage aggregated both by day, and separately [by connection](https://github.com/airbytehq/airbyte-cloud/blob/9bdd59f5495a8b8f743fd8a331214005ab4245ef/billing/src/main/java/io/airbyte/billing/BillingReader.java#L81), so I figured that we would need to support the same thing after switching to the billing vendor.

However, I looked into how the UI uses this endpoint and currently the only part of the return value that is used is [the creditConsumptionByDay data](https://github.com/airbytehq/airbyte/blob/17504e978e77785bc126449478cb0e751c51b413/airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/CreditsUsagePage.tsx#L26). So we can safely remove the `creditConsumptionByConnector` part of the return value of that endpoint, and we can add that functionality back in later if we need it, probably in an endpoint that takes in a connection/source/destination ID as a parameter.
